test/unit/application/workflow/orchestrator.test.ts

```
@@ -21,45 +18,113 @@ class MockSessionService {
   private sessions = new Map<string, any>();
   private stepErrors = new Map<string, Error[]>();
 
+  // Missing properties to match SessionService interface
+  private store: any = null;
+  logger: any = mockLogger;
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)